### PR TITLE
Pull request for display comment popup and choose assets

### PR DESCRIPTION
**Pull request by zuhebahmed88091**
In this branch added->

- Choose images, text, icons, fonts
   - [ ]  Use Font-Awesome for icons
   - [ ]  Font-Family: cursive
   - [ ]  Images from API

- Created a comments popup only with the top section
- When the popup loads, the webapp retrieves data from the API and shows details about the selected item.

### CODE REVIEW

**Review by Fombi-Favour — CHANGES_REQUESTED** [review]
Hello @zuhebahmed88091 
Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
soclose
## Highlights
- All linters checks passed ✔️ 
- Used github flow ✔️ 
## Required Changes ♻️ 
- The **README.md** file is blank. Please kindly fill in the file with the necessary information about this project. 👍
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Fombi-Favour — CHANGES_REQUESTED** [review]
Hello @zuhebahmed88091 
Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
soclose
## Highlights
- All linters checks passed ✔️ 
- Used github flow ✔️ 
## Required Changes ♻️ 
*View the changes in the comments below.*
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1 +1,150 @@
...
+
+As this is one of the first projects of the author, please be kind enough and show your support by DMing on twitter or by contributing to this project. All your efforts are highly appreciated and welcomed!
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+For this project many hours have been invested. For that reason, I would like to thank all my family that supported me in this long journey.
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```
[the remark this answers is not in the document]
Comment (zuhebahmed88091) [reply README.md:150]: Added. Can you check it please?

**Review by Fombi-Favour — APPROVED** [review]
## Status: Approved ✔️ ✔️ ✔️ 
Hello @zuhebahmed88091 
Good job so far! 
Your project is complete now! There's nothing else to say than... it's time to merge it
approved
## Highlights
- All linters checks passed ✔️ 
- Used github flow ✔️ 
- Good documentation in README file ✔️ 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**